controllers/new.go

```diff
@@ -734,6 +802,92 @@ func (k *kubeResources) newPolicyEngineContainer(instance *cv1.UpdateService, im
 	}
 }
 
+func (k *kubeResources) newMetadataContainer(instance *cv1.UpdateService, image string) *corev1.Container {
+	envConfigName := nameEnvConfig(instance)
+	return &corev1.Container{
+		Name:            NameContainerMetadata,
+		Image:           image,
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		Command: []string{
+			"/usr/bin/metadata-helper",
+		},
+		Args: []string{
+			"-vvv",
+			"--signatures.dir",
```

Comment:
Waiting for openshift/cincinnati#877 to merge to get this working, [currently][1] it [gives][2]:

```
2023-11-08T03:43:50.770726034Z error: Found argument '--signatures.dir' which wasn't expected, or isn't valid in this context
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-operator/176/pull-ci-openshift-cincinnati-operator-master-operator-e2e-hypershift-local-graph-data/1722091541649952768
[2]: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cincinnati-operator/176/pull-ci-openshift-cincinnati-operator-master-operator-e2e-hypershift-local-graph-data/1722091541649952768/artifacts/operator-e2e-hypershift-local-graph-data/e2e-test/artifacts/inspect/namespaces/openshift-updateservice/pods/example-5cb95c9497-w828r/metadata/metadata/logs/current.log

Reply:
https://github.com/openshift/cincinnati/pull/877 merged, and this works now.